gpkitmodels/GP/aircraft/wing/gustloading.py

```diff
@@ -32,9 +35,12 @@ def setup(self, static, Wcent, Wwing, V, CL):
 
         constraints = [
             # fit for arctan from 0 to 1, RMS = 0.044
-            FitCS(df, agust, [cosminus1*vgust/V]),
+            V == v,
+            CL == cl,
+            Ww == Wwing,
```

Comment:
why are these necessary?

Reply:
for backwards compatibility so I don't break jho and gassolar

Reply:
I'll fix gassolar and jho once this is done

Reply:
I'd be tempted to make their usage of this more hacky (repeating arguments or w/e) rather than have confusing code in gplibrary...